includes/Settings.php

```diff
@@ -218,6 +218,23 @@ public static function newFromGlobals() {
 		return self::$instance;
 	}
 
+	/**
+	 * @since 3.2
+	 *
+	 * @param array $vars
+	 */
+	public function override( array $vars ) {
```

Comment:
Is this method suppose to be package public? In other words: is it meant to be used by extensions to SMW?

Reply:
> Is this method suppose to be package public? In other words: is it meant to be used by extensions to SMW?

No, it should only be used during `Setup:init`.

Reply:
https://github.com/SemanticMediaWiki/SemanticMediaWiki/pull/4560